scripts/openapi.py

```
@@ -0,0 +1,668 @@
+SCHEMA_ENDPOINT = "/api/schema/"
+
+# ---------------------------------------------------------------------------
+# Core: fetch / load / save
+# ---------------------------------------------------------------------------
+
+
+def fetch_schema(
+    base_url: str,
+    *,
+    token: str | None = None,
+    format: str = "json",  # noqa: A002 — shadows builtin intentionally
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `A002`)

Remove unused `noqa` directive

(RUF100)